hummingbot/connector/exchange/hitbtc/hitbtc_user_stream_tracker.py

```diff
@@ -32,13 +30,13 @@ def logger(cls) -> HummingbotLogger:
 
     def __init__(self,
                  hitbtc_auth: Optional[HitbtcAuth] = None,
-                 trading_pairs: Optional[List[str]] = []):
-        super().__init__()
+                 trading_pairs: Optional[List[str]] = None):
         self._hitbtc_auth: HitbtcAuth = hitbtc_auth
-        self._trading_pairs: List[str] = trading_pairs
-        self._ev_loop: asyncio.events.AbstractEventLoop = asyncio.get_event_loop()
-        self._data_source: Optional[UserStreamTrackerDataSource] = None
-        self._user_stream_tracking_task: Optional[asyncio.Task] = None
+        self._trading_pairs: List[str] = trading_pairs or []
+        super().__init__(data_source=HitbtcAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by `super().__init__`. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.